src/app/counter/counter-booking/counter-booking.component.ts

```diff
@@ -12,6 +12,9 @@ import { Elements, Element, StripeService } from "ngx-stripe";
   styleUrls: ["./counter-booking.component.css"],
 })
 export class CounterBookingComponent implements OnInit {
+  minPrice = 0;
```

Comment:
This number should be in a constants file, if it is not taken from the most expensive flight